src/main/java/com/google/sps/servlets/WaitData.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.servlets;
+
+import java.util.Date;
+import java.util.List;
+
+// Helper class to hold dates and times lists
+public class WaitData {
+  private List<Date> dates;
+  private List<Long> waitTimes;
+
+  public WaitData(List<Date> classDates, List<Long> times) {
```

Comment:
I'm assuming these are two parallel lists. I don't know if there is a good Pair class available to you (I know there's one in google3 but not sure about general Java), but if there is usually I'd prefer that because it's more maintainable.